cmd/repl/repl.go

```diff
@@ -230,6 +239,66 @@ func makeSessionFromEnv() *pb.Session {
 		Submitter:        os.Getenv("SQLFLOW_submitter")}
 }
 
+func switchDatabase(db string, session *pb.Session) {
+	stream := sql.RunSQLProgram("USE "+db, "", session)
+	r := <-stream.ReadAll()
+	switch r.(type) {
+	case string:
+		session.DbConnStr = getDataSource(session.DbConnStr, db)
+		fmt.Println("Database changed to", db)
+		currentDB = db
+	}
+}
+
+func getDatabaseName(datasource string) string {
+	driver, other, e := database.ParseURL(datasource)
+	if e != nil {
+		log.Fatalf("unrecognized data source '%s'", datasource)
+	}
+	re := regexp.MustCompile(`[^/]*/(\w*).*`) // mysql and hive
```

Comment:
We need a comment to explain the purpose of applying the regex here.

Reply:
Done.